boards/chronos/include/periph_conf.h

```diff
@@ -14,17 +15,28 @@
  * @brief       Chronos peripheral configuration
  *
  * @author      Oliver Hahm <oliver.hahm@inria.fr>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
  */
 
-#ifndef PERIPH_CONF_H_
-#define PERIPH_CONF_H_
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
```

Comment:
propably copy+paste. I tend to use guards without trailing underscores in the files I create...